src/vm/exceptionhandling.cpp

```diff
@@ -17,6 +17,10 @@
 #include "eventtrace.h"
 #include "virtualcallstub.h"
 
+#if defined(_TARGET_X86_) || defined(_TARGET_AMD64_)
```

Comment:
@janvorli Thank you for comment. In general, I prefer to remove duplication, and thus 
this PR attempts to remove duplication in ifdef condition, and also make it explicit that the code in ``HandleHardwareException`` is related with the code for ``IsDivByZeroAnIntegerOverflow`` from the following structure:
```
#if defined(_TARGET_X86_) || defined(_TARGET_X64_)
...
#endif 
...
BOOL HandleHardwareException(PAL_SEHException* ex)
{
    ...
#if defined(_TARGET_X86_) || defined(_TARGET_X64_)
    ...
#endif 
    ...
}
```
This code seems to be fine as it as since it is currently only for X86 and X64, and it is unlikely that other architectures needs this code, or I would like to rename ``MAY_THROW_DBZ_ON_OVF`` into some other flag such as ``OVF_IS_SIGANLLED_AS_DBZ``. Could you share your opinion?

Feel free to close this PR if you think that the current code is better.